src/pipecat/services/google/google.py

```diff
@@ -708,6 +712,62 @@ async def _process_context(self, context: OpenAILLMContext):
                                 function_name=c.function_call.name,
                                 arguments=args,
                             )
+                    # Handle grounding metadata
+                    # I believe we are going to receive only one chunk with this information
```

Comment:
I need to double check this. Maybe we are receiving multiples and I need to handle that.

Reply:
I have tested that several times, and It seems only the last chunk that we receive may contain this information.